### DISCUSSION TITLE
Can you please integrate Afterpay?

### DISCUSSION THREAD

**Discussion by BunnyBuddy** [flutter-stripe/flutter_stripe#1240] [Q&A]
We want to implement afterpay in Flutter but couldn't seem to find any guide or package for it.

**Comment by remonh87** [comment]
We support afterpay as in you can confirm an afterpay payment intent. I would recommend following the stripe official docs: https://stripe.com/docs/payments/afterpay-clearpay/accept-a-payment?platform=mobile . Also you can use the ideal payment example as reference [link](https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/screens/regional_payment_methods/ideal_screen.dart).

**Reply by mahimrocky** [reply]
Hi @BunnyBuddy, did you solve that? I'm facing same type of issue "None of the requested payment methods (afterpay_clearpay) match the supported payment types (klarna, cashapp, card)"
Could you please give me a suggestion
